### PR TITLE
Vscode react components branch

### PR DESCRIPTION
**Pull request by irinaponomarevahaulmont**
Please review.

### CODE REVIEW

File: content/modules/vscode-extension/pages/react-component-wizard.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  *  Entity List
+
+  *  Entity Editor
+
+  *  Entity Show.
+
+[[blank-component]]
+=== Blank Component
+
+The *_Blank Component_* template allows us to create a blank component and register it in the application’s menu and in the routing configuration. Use this option is you need to create a page that doesn’t match any other standard component template.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/react-component-wizard.adoc:41]: is you need --> if you need

File: content/modules/vscode-extension/pages/react-component-wizard.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+*_Add to menu_* is the checkbox that specifies whether the created page will be added to the menu.
+
+*_Directory_* is the folder, in which the component will be created.
+
+Now let us take a closer look at the *_Entity List_* wizard and describe the additional parameters.
+
+image::wizard/vscode-entity-list-step-1.png[align=center, width=600]
+
+On the Step 1 the only parameter is *_Query to load items_*, which allows us to select the query tha will be used to load the list of items for the component.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/react-component-wizard.adoc:101]: tha -> that

File: content/modules/vscode-extension/pages/react-component-wizard.adoc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+*_Select arguments to order by_* is the selection of the set of fields, by which sorting will be available on the page.
+
+*_Select argument for pagination_* is the parameter that allows us to choose the argument of the GraphQL query, which will be passing the pagination parameters. If you choose `<none>`, pagination will be turned off for the page. The parameter will be available only if the GraphQL query supports pagination.
+
+All fields supported by the GraphQL endpoint, which was chosen for the loading of the object list from the server, are available for selection in the two above-mentioned fields.
+
+When you have filled in all the fields according to your needs, click *_Create_*.
+
+Once the wizard's work is completed, the extension modifies the React client file and opens the generated component in the code editor.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/react-component-wizard.adoc:123]: if we speak about one file - then should be: extension creates ...
if we speak about several files (one that is being created, and other configuration files that are modified)
then should be: extension modifies React client files ...

File: content/modules/vscode-extension/images/wizard/vscode-created-file.png
Comment (alexbudarov) [inline content/modules/vscode-extension/images/wizard/vscode-created-file.png:1]: please retake this screenshot after you invoke "npm run generate", so the red highlighting of the imports goes away.

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Please review the changes.

**Comment by alexbudarov** [comment]
solve merge conflict pls

**Comment by irinaponomarevahaulmont** [comment]
Resolved, please merge.

**Pull request merged**